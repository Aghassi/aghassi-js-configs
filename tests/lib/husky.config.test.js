const huskyConfig = require('../../lib/husky.config');
const validateConfig = require('./__helpers__/configSnapshot');

// Use the helper suite to generate a test suite to run
validateConfig('husky', huskyConfig);
