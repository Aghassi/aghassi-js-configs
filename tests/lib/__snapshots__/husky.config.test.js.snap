// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`husky config matches default config snapshot 1`] = `
Object {
  "hooks": Object {
    "commit-msg": "commitlint -e $GIT_PARAMS",
    "pre-commit": "lint-staged",
  },
}
`;
